packages/json-file/package.json

```diff
@@ -28,12 +28,12 @@
     "build"
   ],
   "dependencies": {
-    "@babel/code-frame": "~7.10.4",
+    "@babel/code-frame": "^7.18.6",
```

Comment:
This isn't used for transpilation, the code-frame package is pulled in for formatting. This needs thorough testing.